### DISCUSSION TITLE
What is props.navigator ?

### DISCUSSION THREAD

**Discussion by VitaliiTsurkan** [LunatiqueCoder/react-native-media-console#35] [Q&A]
What is props.navigator ?
I use react-navigation.
Could you please provide additional info on how (and what) to pass the navigator?
Should it be props.navigation in react-navigation case ?

**Comment by LunatiqueCoder** [comment]
Hello @VitaliiTsurkan 

https://reactnavigation.org/docs/navigation-prop

You have this navigation object passed through all the screens (not components!) but you can also access it with `useNavigation`: https://reactnavigation.org/docs/use-navigation

I'll move this to a discussion since it's a question and not really an issue.

**Reply by VitaliiTsurkan** [reply]
I have tried exactly as you wrote above and it seems like not working. (

**Reply by LunatiqueCoder** [reply]
@VitaliiTsurkan  I'll try to check it today and come back to you. I will also start working on a new version so in case we find an issue, we will fix in in January most probably.

**Reply by VitaliiTsurkan** [reply]
@criszz77 thank you. When I wrote that it's not working, I mean the fullscreen button and logic are not working only. 
I passed the navigation prop from useNavigation hook like that navigator={navigation}

**Reply by VitaliiTsurkan** [reply]
Also when I press backButton it shows the next error — `"The action 'POP' with payload {"count:" 1} was not handled by any navigator."`
